content/docs/intro/cloud-providers/cloudinit/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title: cloud-init
+meta_desc: This page provides an overview of the cloud-init Provider for Pulumi.
+menu:
+  intro:
+    parent: cloud-providers
+    identifier: clouds-cloudinit
+    weight: 2
+---
+
+The cloud-init provider for Pulumi can be used to render valid MIME-multipart cloud-init config.
+
```

Comment:
Should probably add in a link to the API ref docs to keep this page consistent with the other miscellaneous provider pages like random and TLS.

```suggestion
The cloud-init provider for Pulumi can be used to render valid MIME-multipart cloud-init config.

See the [full API documentation]({{< relref "/docs/reference/pkg/cloudinit" >}}) for complete details of the available cloud-init provider APIs.
```

Reply:
great catch!